src/Dropdown/__tests__/Dropdown-test.js

```diff
@@ -18,6 +18,24 @@ if (React.version.match(/15.[0-5]/)) {
 
 var MockDropdownList = require("./fixtures/MockDropdownList");
 var Dropdown = require("../Dropdown.js");
+var ButtonTrigger = require("../ButtonTrigger.js").default;
+
+class InputTrigger extends ButtonTrigger {
```

Comment:
Is there another way we could implement a mock? Looks like the whole thing belongs more to the `<Select />` pr

Reply:
hm, this isnt related to select in any way?!

Reply:
testing input.value is easier then fetching stuff from the DOM, thats why I extended the trigger here.